tests/Responses/Expected/checkout/reselect_shipping_response.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": @integer@,
+    "items": @array@,
```

Comment:
I'm not sure about it - in case of ``re-`` transitions, the only thing that is relevant for test is checkout state of the order.